meetings/telecon/2022-06-30.md

```diff
@@ -4,4 +4,5 @@
 
 - [selectmenu] Method to change arrow icon? [#548](https://github.com/openui/open-ui/issues/548)
 - [selectmenu] Restricting interactive content in `<selectmenu>` listbox [#540](https://github.com/openui/open-ui/issues/540)
+- [tabs] Panelset & tabs: defining separate goals for each pattern [#559](https://github.com/openui/open-ui/issues/540)
```

Comment:
It looks like you're linking to the wrong issue, can you swap out for this instead?
```suggestion
- [tabs] Panelset & tabs: defining separate goals for each pattern [#559](https://github.com/openui/open-ui/issues/559)
```